packages/rest/.lintstagedrc.js

```diff
@@ -1 +1,4 @@
-module.exports = require('../../.lintstagedrc.json');
+module.exports = {
+	...require('../../.lintstagedrc.json'),
+	'src/**.ts': 'vitest related --run',
```

Comment:
Doesn't this in the root:
`"src/**.ts": "vitest related --run --config ../../vitest.config.ts"` already do the same thing? Since it runs in each file path

Reply:
@ckohen

Reply:
This change was made because the /rest directory has it's own vitest config. If this wasn't included it would use the root config which would break the tests.

Reply:
Then let's be a bit more verbose and include loading the config as an additional parameter instead of it looking in the file-tree